mission_control/navigator_launch/nodes/network_keepalive.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+
+
+class KeepAlive(object):
+    '''
+    Meant to run from any number of remote computers.
+    Make sure the wifi link is somewhere between the computers running the keepalive
+        script and the boat (or else there is no point!)
```

Comment:
Can you add "run on shore, not on the boat", or something along those lines, to make it abundantly clear to new people?